src/client.cr

```diff
@@ -71,7 +71,13 @@ module Crudris
         "author"  => @author_name,
         "content" => content,
       }.to_json
-      return JSON.parse resp.body
+
+      if resp.status_code == 200
+        return JSON.parse(resp.body)
+      else
+        data = JSON.parse(resp.body)["data"] # To make the code more readable.
+        Log.error(exception: Exception.new(data["error"].to_s)) { "Error while creating message." }
```

Comment:
why aren't you raising an error?

Reply:
https://github.com/eludris-community/crudris/pull/2/commits/7dcb3e3ba4b24fea06d4e3f0fc7423ca53dd1d1e
![image](https://user-images.githubusercontent.com/80943664/217525508-283fffc5-3d64-4392-bd84-a5eb286052c5.png)